crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -101,46 +106,69 @@ impl Command {
 			_args.push("--package");
 			_args.push(package)
 		}
-		if args.release {
+		let profile = args.profile.unwrap_or(Profile::Debug);
+		if profile == Profile::Release {
 			_args.push("--release");
+		} else if profile == Profile::Production {
+			_args.push("--profile=production");
 		}
 		cmd("cargo", _args).dir(args.path.unwrap_or_else(|| "./".into())).run()?;
 
-		let mode = if args.release { "RELEASE" } else { "DEBUG" };
-		cli.info(format!("The {project} was built in {mode} mode."))?;
+		cli.info(format!("The {project} was built in {} mode.", profile))?;
 		cli.outro("Build completed successfully!")?;
 		Ok(project)
 	}
 }
 
 #[cfg(test)]
 mod tests {
+	use std::fs;
+	use std::fs::write;
+	use std::path::Path;
 	use super::*;
 	use cli::MockCli;
+	use strum::VariantArray;
+
+	fn add_production_profile(project: &Path) -> anyhow::Result<()> {
```

Comment:
Happy to hear where I can place this function as it now exists in three places.

Reply:
I had the same question in the `pop call contract`. The solution was to have a public function in the crate `pop_contracts` under a testing module that can be then used by the` pop-cli` https://github.com/r0gue-io/pop-cli/pull/306/files#diff-1735e49d7426a2bb8a539485425778554b43ef1ca455b26bd8138b69c69b18aeR27

Reply:
If we see it can be used in contracts then Ale's suggestion is the best approach.

Reply:
It will be used for parachains and builds, not for smart contracts as far as I'm aware. However, I see no reason not to place it in common.

@AlexD10S your suggestion doesn't use any feature flag? This means it is added to the compilation time of the users which doesn't seem ideal. I suggest we come up with a feature name that we can start to use in the repository

Reply:
That is a great suggestion, I will not add it to the templates as default, something we can consider in the future